HACKING.org

```diff
@@ -4,6 +4,8 @@
 - dune subst
 - ocaml tools/gen_version.mlt src/Version.ml M.N.O
 - git add src/Version.ml
+- update CHANGES.md by editing output of tools/gen_changes.sh vCURR vM.N.O
```

Comment:
I don't think we should generate this. These are the PR names, which are often badly written.
We may want to add one more line to explain the feature or add more than 1 contributor (for example, reviewers, reporter, more than one contributor).
Also, we may want to sort entries, for example, bugs fixed, breaking changes, new options, etc...

What do you think ?

Reply:
Yes but automatically generating the list is convenient (it's up to us to have good names for the PRs), and of course we can manually add more details and categorize the items afterwards.

Reply:
Of course we should generate the list now.
But I don't think we should update it at every release but instead in every PRs, by hand.

Reply:
I agree it would be better to do it for each PR (I don't remember how it was done by Josh before the changelog file was removed in the first time), the line in HACKING.org is merely a reminder to keep the file up-to-date before a release.

Reply:
I reverted these changes.